doc/locale/ja/LC_MESSAGES/reference.po

```diff
@@ -28954,6 +28954,68 @@ msgstr ""
 msgid "Outputs reading of token."
 msgstr "トークンの読みを出力します。"
 
+msgid "``TokenPattern``"
+msgstr ""
+
+msgid ""
+"``TokenPattern`` is a tokenizer which is used to extract tokens by regular "
+"expression. This tokenizer extracts only token that matches the specified "
+"regular expression."
+msgstr ""
+"``TokenPattern`` は正規表現でトークンを抽出するのに使われるトークナイザーで"
+"す。このトークナイザーは指定した正規表現にマッチしたトークンだけを抽出しま"
+"す。"
+
+msgid "You can also specify multiple patterns of regular expression."
+msgstr "複数の正規表現を指定することもできます。"
+
+msgid "``TokenPattern`` has optional parameter."
+msgstr "``TokenPattern`` は、省略可能な引数があります。"
+
+msgid "Specify one pattern::"
+msgstr "パターンを1つ指定::"
+
+msgid "Specify multiple patterns::"
+msgstr "複数のパターンを指定::"
+
+msgid "``TokenPattern`` can accept multiple patterns as above."
+msgstr "``TokenPattern`` は上記のように複数のパターンを受け付けます。"
+
+msgid ""
+"Here is an example of ``TokenPattern``. As ``TokenPattern`` only extracts "
+"the token which matches the specified regular expression, it is able to use "
+"like a white list."
+msgstr ""
+"``TokenPattern`` の例です。``TokenPattern`` は指定した正規表現にマッチする"
+"トークンのみを抽出します。そのためホワイトリストみたいなことができます。"
+
+msgid ""
+"For example, let's hide a specific keyword from search results even though "
+"search query contains it."
+msgstr ""
+"例えば、検索するクエリーに含まれていても、特定のキーワードは検索結果から隠す"
+"というのをやってみましょう。"
```

Comment:
In this case, "TokenPattern" looks like that used as blacklists.

Reply:
I've fixed description.

Reply:
Ah...I'm sorry.

In before example, the description was as a whitelist but the example was as a blacklist.
Therefore, in my opinion, we should align the example and description.